dvc/command/version.py

```diff
@@ -46,13 +45,11 @@ def run(self):
             # `dvc config cache.shared group`.
             if os.path.exists(repo.cache.local.cache_dir):
                 info.append(
-                    "Cache: {}".format(self.get_linktype_support_info(repo))
+                    "Cache types: " + self.get_linktype_support_info(repo)
```

Comment:
Let's use `{}` formated string as above with just the addtion of `Cache types:`.